.github/workflows/ci.yml

```diff
@@ -458,6 +458,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -am -pl "${{ matrix.modules }}"
+      - name: Prefetch Ryuk
+        run: |
+          $RETRY docker pull "$(cat testing/trino-testing-containers/src/main/resources/io/trino/testing/containers/testcontainers-ryuk-image.txt)"
```

Comment:
@nineinchnick how i should add this to ci.yml?
where else i should add this?

PTs for sure, others?

Reply:
IIRC @nineinchnick is working on a change that will extract common code to separate step extracted as separate file. Then it will be easy to add such changes